nixos/tests/nginx-http3.nix

```diff
@@ -0,0 +1,145 @@
+import ./make-test-python.nix ({lib, pkgs, ...}:
+let
+  test-certificates = pkgs.runCommandLocal "test-certificates" { } ''
```

Comment:
We should hardcode certificates here, for the reasons explained in https://github.com/NixOS/nixpkgs/pull/91121#discussion_r505410235

Reply:
I did not find a way to execute command `step` in`files` block.

Reply:
Fixed!